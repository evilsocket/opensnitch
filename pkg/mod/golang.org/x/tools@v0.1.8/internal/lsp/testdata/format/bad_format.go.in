package format //@format("package")

import (
	"runtime"
	"fmt"
	"log"
)

func hello() {




	var x int //@diag("x", "compiler", "x declared but not used", "error")
}

func hi() {
	runtime.GOROOT()
	fmt.Printf("")

	log.Printf("")
}
