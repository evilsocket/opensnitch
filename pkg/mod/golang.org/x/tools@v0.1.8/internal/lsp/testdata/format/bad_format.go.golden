-- gofmt --
package format //@format("package")

import (
	"fmt"
	"log"
	"runtime"
)

func hello() {

	var x int //@diag("x", "compiler", "x declared but not used", "error")
}

func hi() {
	runtime.GOROOT()
	fmt.Printf("")

	log.Printf("")
}

